lib/Cli/Cli.js

```diff
@@ -125,6 +125,14 @@ class CLI {
             this.logger.warn('RNA CLI beta version detected.');
         }
         let commands = parse(argv);
+
+        // check if one of the given commands is a non-existing command
+        const wrongCommands = commands.filter((name) => name._ !== '_').some((command) => Object.keys(this.commands).findIndex((cmd) => command._ === cmd) === -1);
```

Comment:
non bastava `commands.filter((name) => name._ !== '_').some((command) => !(command._ in this.commands));` ?

Reply:
fixed